### PR TITLE
DTSSTCI-730

### PR DESCRIPTION
**Pull request by pr00279**
### Change description

Edit error messages so that they are inline with Government guidelines & approved PO changes (approved changes can be seen in the spreadsheet attached to the below ticket)

Please note that Welsh translation is out of scope of this ticket and a separate ticket should be made for Welsh translation

### JIRA link

https://tools.hmcts.net/jira/browse/DTSSTCI-730

### CODE REVIEW

**Review by tomxelliott — APPROVED** [review]

File: src/main/resources/contact-details/translation.json [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
+    "en": {
+      "applicantPhoneNumber": {
+        "required": "Please enter correct mobile phone number",
+        "invalid": "Please enter correct mobile phone number",
+        "atleastOneRequired": "Mobile phone number is empty, please fill up at least one contact"
+      },
+      "applicantHomeNumber": {
+        "required": "Please enter correct home phone number",
+        "invalid": "Please enter correct home phone number",
+        "atleastOneRequired": "Home phone number is empty, please fill up at least one contact"
```
Comment (gmmagruder) [inline src/main/resources/contact-details/translation.json:34]: Please remove all the pleases in the error messages.

Comment (pr00279) [reply src/main/resources/contact-details/translation.json:34]: This is actually not used in the main application - if you run Submit Case from this branch you will never see this text, this is out of the scope of this ticket

File: src/main/resources/representative-details/translation.json [hunk 2]
```diff
@@ -20,26 +22,27 @@
...
-        "required": "Please enter a valid contact number",
-        "invalid": "Please enter a valid contact number"
+        "required": "Enter the contact number of your representative",
+        "invalid": "Enter a telephone number in the correct format"
       },
       "representativeEmailAddress": {
-        "required": "Please enter a valid email address",
-        "invalid": "Every email address consists of 3 elements: local-part, @ symbol (pronounced as “at”), and a domain name"
+        "required": "Enter an email address",
+        "invalid": "Enter an email address in the correct format"
```
Comment (rajaram-nagarajan) [inline src/main/resources/representative-details/translation.json:45]: should we mention with example?
Enter an email address in the correct format, like name@example.com

File: src/main/resources/representative-details/translation.json [hunk 3]
```diff
@@ -20,26 +22,27 @@
...
       "representativeOrganisationName": {
-        "required": "Please enter organisation or business name",
+        "required": "Enter the organisation name or business name of your representative",
         "containsMarkdownLink": "The data entered is not valid for Organisation or business name. Link markdown characters are not allowed in this field."
       },
       "representativeContactNumber": {
-        "required": "Please enter a valid contact number",
-        "invalid": "Please enter a valid contact number"
+        "required": "Enter the contact number of your representative",
+        "invalid": "Enter a telephone number in the correct format"
```
Comment (rajaram-nagarajan) [inline src/main/resources/representative-details/translation.json:41]: should we mention with example?
Enter a telephone number in the correct format, like 01632 960 001 or 07700 900 982

File: src/main/resources/representative-details/translation.json [hunk 4]
```diff
@@ -20,26 +22,27 @@
       "fullNameLabel": "Enw llawn",
       "organisationNameLabel": "Enw’r sefydliad neu’r busnes",
       "contactNumberLabel": "Rhif cyswllt",
-      "emailAddressLabel": "Cyfeiriad e-bost"
+      "contactNumberHint": "Enter a telephone number in the correct format, like 01632 960 001 or 07700 900 982 (WELSH)",
```
Comment (rajaram-nagarajan) [inline src/main/resources/representative-details/translation.json:25]: Why not using the google translator to convert to welsh?

File: src/main/resources/statement-of-truth/translation.json [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
+      "serviceName": "Datganiad Gwirionedd",
+      "statementOfTruthLabel": "Credaf fod y ffeithiau a nodir yn y cais hwn yn wir",
+      "confirmStatement": "Mae hyn yn cadarnhau bod yr wybodaeth yr ydych yn ei chyflwyno yn wir ac yn gywir hyd eithaf eich gwybodaeth. Gelwir hwn yn eich ‘datganiad gwirionedd’.",
+      "continue": "Cyflwyno"
+    }
+  },
+  "errors": {
+    "en": {
+      "applicantStatementOfTruth": {
+        "required": "Please accept the Statement of Truth to continue"
```
Comment (rajaram-nagarajan) [inline src/main/resources/statement-of-truth/translation.json:19]: Should we need to avoid the word **please**?

File: src/main/resources/subject-contact-details/translation.json [hunk 6]
```diff
@@ -1,56 +1,56 @@
...
-                "required" : "Dylech gytuno i’r llys gysylltu â chi gan ddefnyddio’r manylion a ddarparwyd cyn y gallwch barhau"
-            }
-        }
+    "cy": {
+      "continue": "Parhau",
+      "serviceName": "Apelio i’r Tribiwnlys Haen Gyntaf",
+      "title": "Rhowch fanylion cyswllt",
+      "line1": "Rhowch y manylion cyswllt sydd fwyaf addas ar gyfer yr achos hwn.",
+      "emailAddressLabel": "Cyfeiriad e-bost",
+      "emailAddressHint": "",
```
Comment (rajaram-nagarajan) [inline src/main/resources/subject-contact-details/translation.json:21]: Don't we want to give hints in welsh?

File: src/main/resources/representative-details/translation.json (hunk 2 above)
Comment (pr00279) [reply src/main/resources/representative-details/translation.json:45]: The example has been included in the hint and therefore should NOT be included in the error as per government guidelines

File: src/main/resources/representative-details/translation.json (hunk 3 above)
Comment (pr00279) [reply src/main/resources/representative-details/translation.json:41]: The example has been included in the hint and therefore should NOT be included in the error as per government guidelines

File: src/main/resources/representative-details/translation.json (hunk 4 above)
Comment (pr00279) [reply src/main/resources/representative-details/translation.json:25]: Because Welsh translation is out of the scope of this ticket (as declared in the description of this PR), and I do not have a sufficient enough knowledge of Welsh myself to be able to trust that Google Translate provides an accurate enough translation

File: src/main/resources/statement-of-truth/translation.json (hunk 5 above)
Comment (pr00279) [reply src/main/resources/statement-of-truth/translation.json:19]: This is actually not used in the main application - if you run Submit Case from this branch you will never see this text, this is out of the scope of this ticket

File: src/main/resources/subject-contact-details/translation.json (hunk 6 above)
Comment (pr00279) [reply src/main/resources/subject-contact-details/translation.json:21]: Welsh translation is out of the scope of this ticket (as declared in the description of this PR)

File: src/main/resources/upload-appeal-form/translation.json [hunk 7]
```diff
@@ -25,12 +25,12 @@
   },
   "errors": {
     "en": {
-      "SIZE_ERROR": "File size exceeds 100Mb. Please upload a file that is less than 100Mb",
+      "SIZE_ERROR": "File size exceeds the maximum permitted value. Upload a file that is less than 100 MB",
```
Comment (rajaram-nagarajan) [inline src/main/resources/upload-appeal-form/translation.json:28]: Do you think moving the common messages across various translation files to a separate file will help us avoid making the repeated changes in multiple places due to requirement change?

Comment (pr00279) [reply src/main/resources/upload-appeal-form/translation.json:28]: In the process of making a separate ticket to keep recurrent error messages in one single file - will link here once discussed with the BA

Comment (pr00279) [reply src/main/resources/upload-appeal-form/translation.json:28]: https://tools.hmcts.net/jira/browse/DTSSTCI-902 This is the ticket that will address this issue

**Review by tomxelliott — APPROVED** [review]

File: src/main/resources/representative-details/translation.json [hunk 8]
```diff
@@ -20,26 +22,30 @@
       "fullNameLabel": "Enw llawn",
       "organisationNameLabel": "Enw’r sefydliad neu’r busnes",
       "contactNumberLabel": "Rhif cyswllt",
-      "emailAddressLabel": "Cyfeiriad e-bost"
+      "contactNumberHint": "Enter a contact number in the correct format, like 01632 960 001 or 07700 900 982 (WELSH)",
+      "emailAddressLabel": "Cyfeiriad e-bost",
+      "emailAddressHint": "Enter an email address in the correct format, like name@example.com (WELSH)"
```
Comment (gmmagruder) [inline src/main/resources/representative-details/translation.json:27]: Are the welsh translations going to be added in a separate PR?

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Review by gmmagruder — DISMISSED** [review]

File: src/main/resources/upload-appeal-form/translation.json [hunk 9]
```diff
@@ -25,16 +25,16 @@
...
-      "UPLOAD_DELETE_FAIL_ERROR": "Document upload or deletion has failed. Please try again"
+      "SIZE_ERROR": "File size exceeds the maximum permitted value. Upload a file that is less than 100 MB",
+      "FORMAT_ERROR": "This service only accepts files in the formats - MS Word, PDF",
+      "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Delete one of the uploaded files and retry",
+      "CONTINUE_WITHOUT_UPLOAD_ERROR": "You must upload the application before you can proceed further",
+      "NO_FILE_UPLOAD_ERROR": "Choose a file to upload",
+      "UPLOAD_DELETE_FAIL_ERROR": "Document upload or deletion has failed. Try again"
     },
     "cy": {
       "SIZE_ERROR": "Mae maint y ffeil yn fwy na 100Mb. Llwythwch ffeil sy’n llai na 100Mb",
```
[the remark this answers is not in the document]
Comment (pr00279) [reply src/main/resources/upload-appeal-form/translation.json:36]: Yes, we are - as stated above, it will be made into a separate ticket

[the remark this answers is not in the document]
Comment (pr00279) [reply src/main/resources/upload-appeal-form/translation.json:36]: There is also a spreadsheet with the changed error messages attached to the ticket which will be used as the basis for the strings which require a Welsh translation

**Review by oleon23 — DISMISSED** [review]

**Review by oleon23 — APPROVED** [review]

**Review by tomxelliott — APPROVED** [review]
